docs/content/kubernetes-tutorial.md

```diff
@@ -25,13 +25,18 @@ For the purpose of the tutorial we will deploy two policies that ensure:
 
 ## Prerequisites
 
-This tutorial requires Kubernetes 1.13 or later. To run the tutorial locally ensure you start a cluster with Kubernetes version 1.13+, we recommend using [minikube](https://kubernetes.io/docs/getting-started-guides/minikube) or [KIND](https://kind.sigs.k8s.io/).
+This tutorial requires Kubernetes 1.13 or later. To run the tutorial locally ensure you start a cluster with Kubernetes
```

Comment:
1.13 isn't even on this chart anymore: https://endoflife.date/kubernetes

I guess since we're overhauling the content, it would be a good opportunity to try it with a supported kube version? 🤔

Reply:
Updated the recommended kube version to 1.20+.